plugin/src/functionalTest/resources/generateXml/build-with-generated-dsl.gradle

```diff
@@ -0,0 +1,14 @@
+buildscript {
+    dependencies {
+        classpath files(CLASSPATH_STRING)
+    }
+}
+
+apply plugin: 'com.here.jobdsl'
+
+dependencies {
+    compile localGroovy()
+
+    jenkinsPlugins 'org.jenkins-ci.plugins:cloudbees-folder:6.2.1'
+    jenkinsPlugins 'org.jenkins-ci.plugins:cvs:2.13'
```

Comment:
Any particular reason why you chose the CVS plugin?

Reply:
Yes, because it's used in the documentation:
https://github.com/jenkinsci/job-dsl-plugin/wiki/Automatically-Generated-DSL

Reply:
Is that worth a code comment?

Reply:
I have added comments to the groovy files that contain the sample code.